senders/client_factory.go

```diff
@@ -142,6 +148,14 @@ func newWavefrontClient(cfg *configuration) (Sender, error) {
 	return sender, nil
 }
 
+func TracesURL(cfg *configuration) string {
```

Comment:
Could we instead make TracesURL() and MetricsURL() methods of `configuration`?  I think you can still test after making this change.